src/client.ts

```diff
@@ -1,5 +1,6 @@
 import Clova from './types';
 import { Context } from './context';
+import * as express from 'express';
```

Comment:
Can we change this to `import express from 'express'` ?

we can add this `"esModuleInterop": true,` to our current `tsconfig.json` under `compilerOptions`.